doc/releases/changelog-0.21.0.md

```diff
@@ -41,27 +150,15 @@
    DeviceArray([ 0.26043, -0.26043], dtype=float32, weak_type=True))
   ```
   Note that `jax.jit` is not supported for vector-valued QNodes.
-  [(#2110)](https://github.com/PennyLaneAI/pennylane/pull/2110)
 
-* For subclasses of `Operator` where it is known before instantiation, the `num_params` is reverted back to being a
-  static property. This allows to programmatically know the number of parameters before an operator is
-  instantiated without changing the user interface. A test was added to ensure that
-  different ways of defining `num_params` work as expected.
-  [(#2099)](https://github.com/PennyLaneAI/pennylane/pull/2099)
-  [(#2101)](https://github.com/PennyLaneAI/pennylane/pull/2101)
-  [(#2135)](https://github.com/PennyLaneAI/pennylane/pull/2135)
-
-* Development of circuit cutting compiler has begun:
-  A `WireCut` operator has been added for manual wire cut placement
-  when constructing a QNode.
-  [(#2093)](https://github.com/PennyLaneAI/pennylane/pull/2093)
+<h4>Generalized RotosolveOptmizer</h4>
```

Comment:
Might be good to move this one up, I imagine it is something that is quite common to a lot of workflows in PL - if you are optimizing a variational circuit, this will interest you!

Reply:
Sure! Moved to 3. spot.